src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -417,8 +417,18 @@ public partial class View
 
         /// <summary>
         /// An event for visibility change which can be used to subscribe or unsubscribe the event handler.<br />
-        /// This signal is emitted when the visible property of this or a parent view is changed.<br />
+        /// This event is sent when the visibility of this or a parent view is changed.<br />
         /// </summary>
+        /// <remarks>
+        /// <para>
+        /// If VisibilityChangedEventArgs.Type is SELF, then VisibilityChangedEventArgs.Visibility is true if this View's Visibility property is true.
```

Comment:
This sentence is a little awkward, especially on the grammar.
-> If A, B if C.
You mean, 
```suggestion
        /// If VisibilityChangedEventArgs.Type is SELF and  this View's Visibility property is true, then VisibilityChangedEventArgs.Visibility is true.
```
Is it what you want to explained?